tools/patchwork_integration_test.sh

```diff
@@ -0,0 +1,115 @@
+#! /bin/bash
+
+# Author:
+#   Rohit Sarkar <rohitsarkar5398@gmail.com>
+#
+# This work is licensed under the terms of the GNU GPL, version 2.  See
+# the COPYING file in the top-level directory.
+
+set -e
+USAGE=$(cat <<-END
+	$ ./patchwork_integration_test.sh <path-to-public-inbox> <timestamp>
+	Only mails in public inbox since the given <timestamp> will be considered
+	END
+	)
+if [ $# -ne 2 ]
+then
+	echo -e "$USAGE"
+	exit 1
+fi
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+PUBIN_PATH=$1
+SINCE=$2
+MBOX=test-mbox.mbox
```

Comment:
Can we store that under /tmp/$listname.mbox? No need to mess up tools/ with mbox files